content/lang/collections/misc/collmetadata.en.php

```diff
@@ -123,5 +123,7 @@
 $LANG['RECORDID'] = 'recordID';
 $LANG['SAVE_EDITS'] = 'Save Edits';
 $LANG['CREATE_COLL_2'] = 'Create New Collection';
+$LANG['TINYMCE_INFO'] = 'You can access toolbar by pressing key combination ALT + F10 on Windows or OPTION + F10 on MacOS. 
```

Comment:
Because the end user is in the description section when they hear thi text on the screen reader, perhaps it makes sense to say something like, "Collection description. You can access the text editing toolbar for this description by pressing key combination ALT + F10 on Windows or OPTION + F10 on MacOS."